app/src/main/java/pm/gnosis/heimdall/data/remote/models/RelayServiceData.kt

```diff
@@ -58,3 +62,40 @@ data class RelayEstimate(
     @Json(name = "gasPrice")
     val gasPrice: String
 )
+
+@JsonClass(generateAdapter = true)
+data class RelaySafeCreationParams(
+    @Json(name = "owners") val owners: List<Solidity.Address>,
+    @Json(name = "threshold") val threshold: Int,
+    @Json(name = "s") @field:DecimalNumber val s: BigInteger
+)
+
+@JsonClass(generateAdapter = true)
+data class RelaySafeCreation(
+    @Json(name = "signature") val signature: RelaySignature,
+    @Json(name = "tx") val tx: RelaySafeCreationTx,
+    @Json(name = "safe") val safe: Solidity.Address,
+    @Json(name = "payment") val payment: Wei
+)
+
+@JsonClass(generateAdapter = true)
+data class RelaySignature(
```

Comment:
Why not use `ServiceSignature`?

Reply:
Different service thus separated the payloads.